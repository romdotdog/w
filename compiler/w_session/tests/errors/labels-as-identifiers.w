fn main() {
    // let $label = 1;
    $label
}