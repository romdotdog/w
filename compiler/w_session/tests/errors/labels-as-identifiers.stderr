tests/errors/labels-as-identifiers.w:4:1: error: ':' expected here
}
^
tests/errors/labels-as-identifiers.w:4:1: error: only a loop or block can follow a label
}
^
